Formula/ldid-procursus.rb

```diff
@@ -0,0 +1,25 @@
+class LdidProcursus < Formula
+  desc "Put real or fake signatures in a Mach-O binary"
+  homepage "https://github.com/ProcursusTeam/ldid"
+  url "https://github.com/ProcursusTeam/ldid.git",
+     tag:      "v2.1.5-procursus7",
+     revision: "aaf8f23d7975ecdb8e77e3a8f22253e0a2352cef"
+  version "2.1.5-procursus7"
```

Comment:
is "procursus" really a part of the version?

Reply:
Yes.